src/components/BankTransactions/BankTransactions.tsx

```diff
@@ -318,61 +324,67 @@ const BankTransactionsContent = ({
-          bankTransactions={bankTransactions}
-          editable={editable}
-          removeTransaction={removeTransaction}
-          containerWidth={containerWidth}
-          stringOverrides={stringOverrides?.bankTransactionCTAs}
-          showDescriptions={showDescriptions}
-          showReceiptUploads={showReceiptUploads}
-          showTooltips={showTooltips}
-        />
-      ) : null}
-
-      {!isLoading && listView && mobileComponent === 'mobileList' ? (
-        <BankTransactionMobileList
-          bankTransactions={bankTransactions}
-          editable={editable}
-          mode={mode}
-          removeTransaction={removeTransaction}
-          initialLoad={initialLoad}
-          showTooltips={showTooltips}
-          showReceiptUploads={showReceiptUploads}
-          showDescriptions={showDescriptions}
-        />
-      ) : null}
+      {!isLoading && listView && mobileComponent !== 'mobileList'
+        ? (
+          <BankTransactionList
+            mode={mode}
+            bankTransactions={bankTransactions}
+            editable={editable}
+            removeTransaction={removeTransaction}
+            containerWidth={containerWidth}
+            stringOverrides={stringOverrides?.bankTransactionCTAs}
+            showDescriptions={showDescriptions}
+            showReceiptUploads={showReceiptUploads}
+            showTooltips={showTooltips}
+          />
+        )
+        : null}
 
-      {listView && isLoading ? (
-        <div className='Layer__bank-transactions__list-loader'>
-          <Loader />
-        </div>
-      ) : null}
-
-      {!isSyncing || listView ? (
-        <DataStates
-          bankTransactions={bankTransactions}
-          isLoading={isLoading}
-          isValidating={isValidating}
-          error={error}
-          refetch={refetch}
-          editable={editable}
-        />
-      ) : null}
+      {!isLoading && listView && mobileComponent === 'mobileList'
+        ? (
+          <BankTransactionMobileList
+            bankTransactions={bankTransactions}
+            editable={editable}
+            mode={mode}
+            removeTransaction={removeTransaction}
+            initialLoad={initialLoad}
+            showTooltips={showTooltips}
+            showReceiptUploads={showReceiptUploads}
+            showDescriptions={showDescriptions}
+          />
+        )
+        : null}
+
+      {listView && isLoading
+        ? (
+          <div className='Layer__bank-transactions__list-loader'>
+            <Loader />
+          </div>
+        )
+        : null}
+
+      {!isSyncing || listView
+        ? (
+          <DataStates
+            bankTransactions={bankTransactions}
+            isLoading={isLoading}
+            isValidating={isValidating}
+            error={error}
+            refetch={refetch}
+            editable={editable}
+          />
+        )
+        : null}
 
       {!monthlyView && (
-        <div className='Layer__bank-transactions__pagination'>
```

Comment:
Removing `<div className='Layer__bank-transactions__pagination'>` is actual change in this file. Other changes are lint fixes